src/chrome/content/rules/Torrentz.to.xml

```diff
@@ -1,26 +1,10 @@
-<!--
-	Insecure cookies are set for these hosts:
-
-		- torrentz.to
-		- www.torrentz.to
-
--->
 <ruleset name="Torrentz.to">
 
-	<!--	Direct rewrites:
-				-->
 	<target host="torrentz.to" />
 	<target host="www.torrentz.to" />
```

Comment:
mismatch/expired on dev.torrentz.to (redirects to (www.)torrentz-pro.net which would work)